### ISSUE TITLE
Feature request: remove {{uw- prefix from warning menu

### ISSUE THREAD

**Issue by Gaelan** [azatoth/twinkle#491]
Often, I know what warning I want, but spend several seconds trying to find it in the warning menu. If the entry in the menu simply started "test1" instead of "{{uw-test1}}" (for example), I could just click the menu and start typing "test" to find the entry. OTOH, it makes the mapping to actual templates less clear, so I'm not sure it if's worth that confusion.

**Comment by QEDK** [comment]
Now that I think, is there a particular reason for having template formats at all? Welcome has it, Talkback doesn't, Tag does and CSD doesn't.
@MusikAnimal @atlight

**Comment by Amorymeltzer** [comment]
@QEDK Welcome, TB, Tag, and CSD all don't have menus, so there's no harm there; in those cases I think having the `{{whatever}}` reinforces that yes, this is a specific template you are adding?  That makes sense to me, except for CSD where the typical thinking process is (mostly) by category category rather than template.

I think that's what's happening here.  One the reason it's useful in warn is for custom tags, especially with custom parameters (e.g. `{{User:Jimbo/warn1|hello=wazzup}}`), so that it's clear what you're doing.  In the case of warnings, where everything else begins with `{{uw-`, I would agree that the added benefit of hitting the relevant key would be more valuable.

tldr: I'd be in favor of this (removing the formatting from the warn menu) but would also be in favor of adding it to talkback.

**Comment by siddharthvp** [comment]
For a much superior user experience in selecting a template from a long list, we could use [jquery.chosen](https://harvesthq.github.io/chosen/), which provides a search box within the select menu. It can be loaded as `mw.loader.load("jquery.chosen");` This is also used in the AFCH script, from where I got the idea.

**Comment by Amorymeltzer** [comment]
#517 opened.

One thing I considered but didn't do was to change the major menu names to put the numbers first.  That is, change `General note (1)` to `1: General note` and so on.  That would make the whole menu very keyboard-accessible to use, negating the need for the cursor for most actions.  It didn't look particularly great imo, but maybe it's worth it.

**Comment by Amorymeltzer** [comment]
A few folks on IRC said they'd find it useful, so I've pushed a commit in case anyone wants to check it out.
